### DISCUSSION TITLE
Why did I add two enzymes in individual and not in the processes of drug, and PK changed?

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1261] [Questions & Problems]
I added CYP2C9 and UGT enzymes to individual, but my drugs are not metabolized by these two enzymes, and I have not set them in processes (the settings in simulations have not changed). Why does the PK of drugs decrease？

Thank you in advance :)

**Comment by msevestre** [comment]
Processes are mapped automatically by name in the simulation settings. Are you *sure* that you didn't activate them without knowing

**Reply by noxavier** [reply]
I'm sure the settings in simulations have not changed.That is a model file I downloaded, and I found that changing the name of the enzyme in the file does not change the value. The simulated PK will also change.

**Comment by msevestre** [comment]
Can you share the project file? Depending on the version used, some differences can be expected, for instance when an error was fixed in some of the model parameters, dependencies etc ..
Running the simulation as is should not change anything. But recreating the simulation will use the *new* structure and therefore, potentially, lead to changes

**Reply by noxavier** [reply]
https://github.com/Open-Systems-Pharmacology/Repaglinide-Model   
Thanks！

**Reply by msevestre** [reply]
ok this is a model that was created with version 7.3 of PK-Sim. A lot has changed, bugs were corrected. Not surprising here to see a slight differences when recreating the model.

**Reply by noxavier** [reply]
So,how to solve the problems of different versions?

**Reply by msevestre** [reply]
Are the results significantly different?

**Reply by noxavier** [reply]
All AUCs in the model have been reduced. For example, this is a change that only renames of one enzyme and keeps the other contents unchanged. Therefore, according to the change of PK SIM version, what parameters should I change to keep the original state.
![图片1](https://user-images.githubusercontent.com/109507330/180911737-02ea0950-0cd1-4db8-96fa-867a4fee9282.jpg)

**Reply by msevestre** [reply]
That's a significant change.... something is definitely not ok. I am not sure at that stage

**Reply by StephanSchaller** [reply]
@noxavier ,  you are now using Version 11?
When renaming are you also updating the simulations from the building blocks? Please describe in more detail what you did so we can reproduce the observed outcome. Thanks!

@msevestre, could this not be related to changes in the protein expression interface from the upgrade V9-->V10?

**Reply by StephanSchaller** [reply]
@noxavier, @msevestre, it is enough to re-configure (without changes) and re-run the simulation 
![image](https://user-images.githubusercontent.com/25477643/181101935-8a4930bd-d6bb-4962-a80d-3fa7f72d96ef.png)
in V11, to get the changes, so I am quite certain this is due to the change in the expression data interface.

@PavelBal , do we have the guide on how to update expression data from older (< V10) models somewhere online?

**Reply by msevestre** [reply]
This is only true if we do the same with v9 and there are no changes. I thought the changes were only for transporters but @noxavier is speaking only of enzymes...

**Reply by StephanSchaller** [reply]
I tested it, no changes in V9 and yes, there are transporters:
![image](https://user-images.githubusercontent.com/25477643/181106745-6d8db02f-8459-4a4e-8836-6bc9d6471024.png)

**Reply by msevestre** [reply]
ok then we have it...

**Reply by noxavier** [reply]
@StephanSchaller V11 adds expression profiles. When I make any changes to the ADME in the model, clicking "update from building blocks" will automatically load the contents of expression profiles and need to reconfigure simulation. The PK obtained in this way must change.

**Reply by msevestre** [reply]
@noxavier This is not what @StephanSchaller meant I believe. With v10 of PK-Sim, the implementation of relative expressions was modified and is not directly compatible. Some models need to be fitted again unfortunately

**Reply by noxavier** [reply]
ok, thanks!

**Reply by noxavier** [reply]
> @noxavier This is not what @StephanSchaller meant I believe. With v10 of PK-Sim, the implementation of relative expressions was modified and is not directly compatible. Some models need to be fitted again unfortunately

Can you explain in detail what relative expressions have been modified in V10 to cause incompatible?

**Comment by PavelBal** [comment]
> @PavelBal , do we have the guide on how to update expression data from older (< V10) models somewhere online?

 Nothing online. We could think of creating a section of documentation from some internal material.

**Reply by StephanSchaller** [reply]
@PavelBal I guess there is nothing online, yet, right?

**Reply by StephanSchaller** [reply]
@sfrechen, do you have a documentation easily transferrable to Wiki style or the OSP documentation?
